### PR TITLE
testing/py-pandas: new aport

### PR DESCRIPTION
**Pull request by cgiraldo**
http://pandas.pydata.org/
Powerful data structures for data analysis, time series, and statistics
testing/

### CODE REVIEW

**Comment by cgiraldo** [comment]
Please, 
Review current testing/py-pandas of the repo, because it gives an ERROR: py-pandas-0.23.4-r0: BAD signature when installing.

Also the APK of the repo does not generate independent py2-pandas and py3-pandas packages.

The  APK of this PR generate py2-pandas and py3-pandas, so we can install pandas for python3 without depending on python2.

**Comment by ncopa** [comment]
Do we really need python2 support? I don't want add python2 packages unless specifically needed.

**Comment by cgiraldo** [comment]
That's my point. This APKBUILD provides both variants as py2- and py3- subpackages, so you can add only the corresponding pandas package and its dependencis for python3 or python2.

The APKBUILD I find in the current Alpine testing repository provides a single package that installs python2 with it, and I think is not a desired behavior.

File: testing/py-pandas/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+	local python; for python in python2 python3; do
+		$python setup.py install --prefix=/usr --root="$pkgdir"
+	done
+
+	install -D -m 644 LICENSE \
+		"$pkgdir"/usr/share/licenses/$pkgname/LICENSE
+}
+
+package() {
```
Comment (Ikke) [inline testing/py-pandas/APKBUILD]: Package function is defined twice

Comment (cgiraldo) [reply testing/py-pandas/APKBUILD]: Removed duplication.

**Comment by cgiraldo** [comment]
I check the alpine package:
[https://pkgs.alpinelinux.org/package/edge/testing/x86_64/py-pandas](https://pkgs.alpinelinux.org/package/edge/testing/x86_64/py-pandas)

And the package does not present subpackages py3-pandas nor py2-pandas. 

But when build the package locally, It generates all three packages py-pandas py2-pandas and py3-pandas packages.

Maybe a problem with the APKBUILD definition?

**Comment by Ikke** [comment]
@cgiraldo Yes, ncopa looked into it earlier. `subpkgarch` is an internal variable and should not be used in APKBUILDs. This is what is preventing from the subpackages to be added to the repos, causing it to be rebuild over and over.

**Comment by Ikke** [comment]
I'll push a fix for this (verifying first whether it works).

**Comment by Ikke** [comment]
The fix has been pushed and it now builds correctly.

**Pull request closed without merging**